src/applications/vaos/components/ConfirmedAppointmentListItem.jsx

```diff
@@ -1,53 +1,85 @@
 import React from 'react';
-import { Link } from 'react-router';
 import {
-  getAppointmentId,
+  getClinicName,
   getAppointmentTitle,
   getAppointmentLocation,
   getAppointmentDateTime,
   isVideoVisit,
+  isCommunityCare,
 } from '../utils/appointment';
+import {
+  CANCELLED_APPOINTMENT_SET,
+  APPOINTMENT_TYPES,
+} from '../utils/constants';
 import VideoVisitLink from './VideoVisitLink';
 
 export default function ConfirmedAppointmentListItem({
   appointment,
+  type,
   cancelAppointment,
 }) {
-  return (
-    <li className="vads-u-border-left--5px vads-u-border-color--green vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-bottom--3">
-      <h2 className="vads-u-margin--0 vads-u-margin-bottom--2p5 vads-u-font-size--md">
-        {getAppointmentTitle(appointment)}
-      </h2>
+  let canceled = false;
+  if (type === APPOINTMENT_TYPES.vaAppointment) {
+    canceled = CANCELLED_APPOINTMENT_SET.has(
+      appointment.vdsAppointments?.[0].currentStatus,
+    );
+  }
 
-      <div className="vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row">
-        <div className="vads-u-flex--1 vads-u-margin-bottom--1p5">
-          <h3 className="vads-u-margin--0 vads-u-margin-bottom--1 vads-u-font-size--base vads-u-font-family--sans">
-            {' '}
-            When{' '}
-          </h3>
-          {getAppointmentDateTime(appointment)}
-        </div>
-        <div className="vads-u-flex--1 vads-u-margin-bottom--1p5">
-          <h3 className="vads-u-margin--0 vads-u-margin-bottom--1 vads-u-font-size--base vads-u-font-family--sans">
-            {' '}
-            Where{' '}
-          </h3>
-          {isVideoVisit(appointment) ? (
-            <VideoVisitLink appointment={appointment} />
+  return (
+    <li className="vads-u-background-color--gray-lightest vads-u-padding--2p5 vads-u-margin-bottom--3">
+      <div className="vads-u-display--flex vads-u-justify-content--space-between">
+        <div className="vaos-appts__status vads-u-flex--1">
+          {canceled ? (
+            <i className="fas fa-exclamation-circle vads-u-color--secondary-dark" />
           ) : (
-            getAppointmentLocation(appointment)
+            <i className="fas fa-check-circle vads-u-color--green" />
           )}
+          <span className="vads-u-font-weight--bold vads-u-margin-left--1 vads-u-display--inline-block">
+            {canceled ? 'Canceled' : 'Confirmed'}
+          </span>
         </div>
+
+        {!canceled && (
```

Comment:
I don't think we can cancel CC appointments, so might want to add that to the logic here.